### ISSUE TITLE
Suggestion - add inking for your nav!

### ISSUE THREAD

**Issue by jzhu01** [Modelsson/tea_cozy#5]
Seen here: https://github.com/Modelsson/tea_cozy/blob/master/index.html#L21-L23

it's actually possible to link your nav elements to specific IDs on a page! I recommend you try it out - here's a link for reference: http://www.webweaver.nu/html-tips/link-within-a-page.shtml

**Comment by Modelsson** [comment]
Yeah, I know how to do that. But I saw no point in actually linking the nav this time, since everything was on one page and very close together.
